suite-native/device/src/hooks/useDetectDeviceError.tsx

```diff
@@ -58,27 +57,27 @@ export const useDetectDeviceError = () => {
     useEffect(() => {
         if (isUnacquiredDevice) {
             showAlert({
-                title: translate('moduleDevice.unacquiredDeviceModal.title'),
-                description: translate('moduleDevice.unacquiredDeviceModal.description'),
+                title: <Translation id="moduleDevice.unacquiredDeviceModal.title" />,
```

Comment:
not sure if I like it here, plain `translate` seems more appropriate

Reply:
title prop on showAlert was not changed and it was already ReactNode before this PR. So far the alert content was not stylable and we were not able to use this for various text styles and we had to use different components in that case or change the design to make it work with string with one style only.